src/main/java/micdoodle8/mods/galacticraft/core/util/GCCoreUtil.java

```diff
@@ -98,14 +95,8 @@ public static int getNextValidEggID() {
         return eggID;
     }
 
-    @SuppressWarnings("deprecation")
     public static void registerGalacticraftNonMobEntity(Class<? extends Entity> var0, String var1, int trackingDistance,
             int updateFreq, boolean sendVel) {
-        if (FMLCommonHandler.instance().getEffectiveSide() == Side.CLIENT) {
-            LanguageRegistry.instance().addStringLocalization(
-                    "entity.GalacticraftCore." + var1 + ".name",
-                    GCCoreUtil.translate("entity." + var1 + ".name"));
-        }
```

Comment:
Should also remve it here:
https://github.com/GTNewHorizons/Galacticraft/blob/5e16cfe281b64cbc86666f8ce41e943384c51705/src/main/java/micdoodle8/mods/galacticraft/planets/mars/MarsModule.java#L291-L293